.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
+
+# Install Homebrew For Linux
+RUN /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" && \
+    eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)" && \
+    echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >> /home/vscode/.zshrc && \
+    echo "eval \"\$($(brew --prefix)/bin/brew shellenv)\"" >> /home/vscode/.bashrc && \
+    # Install Graphite
+    brew install withgraphite/tap/graphite
```

Comment:
(question) Are there any installation methods that use apt, pip, or npm? Would be nice to not have another package manage system.

Reply:
Makes sense! Yeah they recommended brew (https://graphite.dev/docs/install-the-cli), but I agree with that and npm seems to work the same! Created https://github.com/civiform/civiform/pull/7110 which installs with npm